### PR TITLE
fix: Set toolbar direction based on toolbar language

### PR DESCRIPTION
**Pull request by fsbraun**
## Description

@sakhawy 

This PR sets the toolbar text direction based on the toolbar language and not based on the document language.

This needs a "polyfill" to work with Chrome before 120 which is added at least in a basic version. 

rtl

## Related resources

* #...
* #...

## Checklist

* [x] I have opened this pull request against ``develop-4``
* [ ] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on [Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

File: cms/static/cms/sass/components/_toolbar.scss [hunk 1]
```diff
@@ -241,20 +241,14 @@
 
     .cms-toolbar-item {
         float: none;
-        [dir=rtl] & {
```
Comment (sakhawy) [inline cms/static/cms/sass/components/_toolbar.scss:244]: As the comments say, if you didn't set `float` to `none`, if you're on mobile, the `cms-toolbar-item` will float to the right and won't be clickable:
image

**Review by sakhawy — CHANGES_REQUESTED** [review]
Aside from the small comment, LGTM! I tested it on a <120 chrome and the rtl patch is working perfectly!

**Comment by fsbraun** [comment]
The patch is minimal, though. We might find places with weird layout. I'll try to add a few more lines later.

**Comment by sakhawy** [comment]
Alright, I'll do some manual tests later!

**Comment by fsbraun** [comment]
@sakhawy So, the polyfill should be complete ;-)

**Comment by fsbraun** [comment]
Once, we have this merged, I'll chery-pick it into #7901

**Comment by fsbraun** [comment]
@sakhawy Can you give this a final check, so we can merge it. I would love to backport it to 3.11 and include it in 3.11.6 which I am about to release...

File: cms/static/cms/sass/components/_modal.scss [hunk 2]
```diff
@@ -193,7 +193,7 @@
 
 .cms-modal-resize {
```
Comment (sakhawy) [inline cms/static/cms/sass/components/_modal.scss:194]: I think the resize cursor should also be flipped (and patched in `_rtl_patch.scss`):

```css
    &:dir(rtl) {
        cursor: ne-resize;  // cursor pointing north-east
    }
```

File: cms/static/cms/sass/components/_modal.scss [hunk 3]
```diff
@@ -206,6 +206,9 @@
         bottom: 5px;
         right: 5px;
         font-size: $font-size-small;
+        &:dir(rtl) {
+            transform: scale(-1,1) translate(2px, 0);  // flip drag triangle
```
Comment (sakhawy) [inline cms/static/cms/sass/components/_modal.scss:210]: I don't think this is patched in `_rtl_patch.scss`

**Review by sakhawy — CHANGES_REQUESTED** [review]
I tested the changes on Firefox 125 and Chrome 110. I also compared it against django-cms 4.1.1. Aside from the small changes, LGTM!

**Comment by fsbraun** [comment]
Well spotted, thank you!

**Comment by fsbraun** [comment]
@sakhawy Fixed both resize cursor and missing polyfill patch. If you're good to go, let me know by approving the PR :-)

File: cms/static/cms/sass/libs/_rtl_patch.scss [hunk 4]
```diff
@@ -41,6 +41,13 @@
     float: right;
 }
 
+.cms-modal-resize {
+    cursor: nw-resize;
```
Comment (sakhawy) [inline cms/static/cms/sass/libs/_rtl_patch.scss:45]: Just a tiny change, this should be "n**e**-resize"

**Review by sakhawy — APPROVED** [review]
LGMT, thanks!

**Review by marksweb — APPROVED** [review]
Happy to approve with Moe's approval in place 👏

**Pull request merged**